src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -231,6 +249,22 @@ private void LoadItems(PackageItemListViewModel selectedPackageItem, Cancellatio
                     _loadingStatusBar.SetError();
                     _loadingStatusBar.Visibility = Visibility.Visible;
                 }
+                finally
+                {
+                    if (_loadingStatusIndicator.Status != LoadingStatus.NoItemsFound
```

Comment:
if logic needs to be improved. think you want this to be an &&

Reply:
Yes. I want to execute the the code block if and only if status is not `NoItemsFound` neither `ErrorOccurred`